contributors/guide/README.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Kubernetes Contributor Guide"
+title: "Getting Started with Contributing"
 weight: 1
-slug: "guide" 
+slug: "getting-started-with-contributing" 
```

Comment:
```suggestion
slug: "guide" 
```
The slug should still be guide for this. As the readme it will default to `_index.md` and take on the role of the parent dir (`guide`)